app.yaml.template

```diff
@@ -0,0 +1,33 @@
+# Google App Engine Config
+
+runtime: python37
+
+# The --http flag exposes the app with uWSGI only and will act as the router/proxy/load balancer
+# If we want an nginx webserver upstream for more features, we would have to use a --socket flag so that nginx can communicate with uWSGI
+# More info here: https://uwsgi-docs.readthedocs.io/en/latest/tutorials/Django_and_nginx.html
+entrypoint: uwsgi --http :8080 --module tcf_core.wsgi --master --processes 1 --threads 2
+
+handlers:
+# This configures Google App Engine to serve the files in the app's static
+# directory.
+- url: /static
+  static_dir: tcf_website/static/
+
+# This handler routes all requests not caught above to your main app. It is
+# required when static routes are defined, but can be omitted (along with
+# the entire handlers section) when there are no static files defined.
+- url: /.*
```

Comment:
should we add a `secure: always` to force https? https://stackoverflow.com/questions/5367974/https-only-in-google-app-engine

Reply:
Added!